test/tests/tsdx-build-closure-compiler.test.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @jest-environment node
+ */
+
+const shell = require('shelljs');
+const util = require('../fixtures/util');
+
+shell.config.silent = false;
+
+const grep = (pattern, file) => {
+  const output = shell.grep(pattern, file);
+  const grepOutput = output.stdout.replace(`\n`, ``);
```

Comment:
Should have a comment here on why this is necessary, e.g. `// shelljs.grep always has at least one newline`. But the `match` code you used before is probably better than this anyway.

Reply:
fully agree)